clink_dev/src/main/java/com/feature/LibfgTransferOp.java

```diff
@@ -0,0 +1,64 @@
+package com.feature;
+
+import com.alibaba.alink.operator.batch.BatchOperator;
+import com.sun.jna.Memory;
+import com.sun.jna.Native;
+import com.sun.jna.Pointer;
+import com.sun.jna.ptr.PointerByReference;
+import org.apache.flink.api.common.typeinfo.TypeInformation;
+import org.apache.flink.api.common.typeinfo.Types;
+import org.apache.flink.api.java.DataSet;
+import org.apache.flink.ml.api.misc.param.Params;
+import org.apache.flink.table.api.TableSchema;
+import org.apache.flink.types.Row;
+
+import java.io.Serializable;
+
+public class LibfgTransferOp extends BatchOperator<LibfgTransferOp>
+        implements LibfgTransferParams<LibfgTransferOp>, Serializable {
+
+    private static final String libfgSoPath = "/flink/usrlib/libperception_feature_plugin.so";
```

Comment:
IMO, the libfg so file shouldn't have a default value or a fixed value specified.

Reply:
Agree, fixed it in the new commit.